prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+CP1404/CP5632 Practical
+Counting the different words in a user string input
+"""
+
+word_to_count = {}
+
+user_string = input("Input String: ")
+words = user_string.split()
+for word in words:
+    if word in word_to_count:
+        word_to_count[word] += 1
+    else:
+        word_to_count[word] = 1
+
+words = list(word_to_count.keys())
+words.sort()
+
+max_length = len(max(word_to_count, key=len))
```

Comment:
You could have also used concatenation here like man((len(word) for word in words)) it would do a pretty similar job